scripts/cli/TimerMode.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2022 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoQtiTest\scripts\cli;
+
+use oat\oatbox\extension\script\ScriptAction;
+use oat\oatbox\reporting\Report;
+
+class TimerMode extends ScriptAction
+{
+    public const SET_TIMER_MODE = 'set';
+    public const SUPPORTED_MODES = [
+        'server',
+        'client'
+    ];
+
+    protected function provideOptions()
+    {
+        return [
+            self::SET_TIMER_MODE => [
+                'prefix' => 's',
+                'longPrefix' => 'set',
+                'cast' => 'string',
+                'flag' => false,
+                'required' => false,
```

Comment:
We can also make it required

Reply:
Yes, it was the case on the first version, then I decided to also give the possibility to display the current value without dong any change by calling the script without argument.

I'll add a comment to highlight this